src/components/Categories.js

```diff
@@ -1,12 +1,50 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import Header from './Header';
 import './Book.css';
+// import { useDispatch } from 'react-redux';
+// import { status } from '../redux/categories/categories';
 
-const Categories = () => (
-  <>
-    <Header />
-    <button className="book-button" type="button">Check Status</button>
-  </>
-);
+import { checkStatus } from '../redux/categories/categories';
+
+// const Categories = () => {
+//   const status = useSelector((state) => state.categReducers);
+//   const dispatch = useDispatch();
+//     <>
+//       <Header />
+//       <button className="book-button" type="button" onClick={() =>
+//  { dispatch(checkStatus()); }}>Check Status</button>
+//       <p>{status}</p>
+//     </>;
+// };
+
+function Categories() {
+  const status = useSelector((state) => state.categ);
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <Header />
+      <button className="book-button" type="button" onClick={() => { dispatch(checkStatus()); }}>Check Status</button>
```

Comment:
Please ensure that the `checkCategoriesStatus` dispatches the `underConstruction` action creator to the screen. At the moment, nothing shows on the screen. Kindly fix that.